assets/base.css

```diff
@@ -1449,7 +1449,15 @@ details[open] > .share-button__fallback {
   z-index: 3;
   width: 100%;
   min-width: max-content;
-  box-shadow: 0 0 0 0.1rem rgba(var(--color-foreground), 0.55);
+  border: 0.1rem solid transparent;
+  border-radius: var(--inputs-radius);
+  box-shadow: 0 0 0 var(--inputs-border-width) rgba(var(--color-foreground), var(--inputs-border-opacity));
+  filter: drop-shadow(
+    var(--inputs-shadow-horizontal-offset)
+    var(--inputs-shadow-vertical-offset)
+    var(--inputs-shadow-blur-radius)
+    rgba(var(--color-foreground), var(--inputs-shadow-opacity))
+  );
```

Comment:
The shareback button focus gets partially hidden. Adding a z-index on focus could help. 
<img src="https://screenshot.click/13-21-rztak-ewkfn.png" with="350" />

Reply:
Also, the input could receive an overwrite not to have a radius to the right: ![](https://screenshot.click/22-01-h433t-08uwn.png)

Reply:
@ludoboludo I believe it makes sense to keep it rounded and affected by the corner radius for consistency 🤔 @melissaperreault what do you think?

Reply:
If we are though, let's also have roundness to the copy icon focus https://screenshot.click/22-01-hn3iy-g6cz5.gif